README.md

```diff
@@ -50,6 +50,13 @@ If you'd like to set a global context for your process, you can do the following
 contextual_logger.global_context = { service_name: 'test_service' }
 ```
 
+#### Redaction
+In order to register sensitive strings to the logger for redaction to occur, do the following:
+```ruby
+contextual_logger.redactor.match('some_sensitive_data')
+```
+This string will then be replaced with `<redacted>` whenever the logger is about to log to the file.
```

Comment:
Will the key still be shown, but `<redacted>` will replace the value?

Reply:
I updated the readme to reflect the usecase better.